_posts/2016-04-27-introducing-infractores.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Introducing Infractores"
+date: 2016-04-26 10:37:00
+categories: ["open-source", "crowdsourcing"]
+author: "etagwerker"
+---
+
+I've always been a big fan of [scratching your own itch](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar#Lessons_for_creating_good_open_source_software). My latest itch was the **insane amount of parking
+infractions** that I see everyday in **Buenos Aires**, near [our office](https://foursquare.com/v/ombulabs-hq/52f0e47311d25da04d101b62).
+
+We decided to build a **simple tool** that would allow anyone with a [Twitter](https://twitter.com) account to *report a parking infraction*. You just
```

Comment:
@etagwerker I think it would sound a little better if you added worded it like: All you to do is 
submit a geolocation enabled tweet and a couple of
 photos as (**evidence!**)